challenge_serving_drinks/src/challenge_serving_drinks/drive_in.py

```diff
@@ -0,0 +1,88 @@
+# ROS
+import smach
+
+# Robot smach states
+import robot_smach_states as states
+from robocup_knowledge import knowledge_loader
+
+# Serving drinks
+from .sd_states import AskAvailability
+
+# Knowledge
+CHALLENGE_KNOWLEDGE = knowledge_loader.load_knowledge("challenge_serving_drinks")
```

Comment:
Don't use capital letters, it is used for global constants.

Reply:
Will be reflected in the new PR (styling)